packages/core/src/invocation.js

```diff
@@ -8,6 +9,30 @@ import { delegate } from './delegation.js'
  */
 export const invoke = options => new IssuedInvocation(options)
 
+/**
```

Comment:
```suggestion
/**
 * Takes a link of the `root` block and a map of blocks and constructs an
 * `Invocation` from it. If `root` is not included in the provided blocks it
 * throws an error. If root points to wrong block (that is not an invocation)
 * it will misbehave and likely throw some errors on field access.
 *
```